install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
 $ ansible-playbook <OPENSHIFT_ANSIBLE_DIR>/byo/openshift-cluster/openshift-metrics.yml \
    -e openshift_metrics_install_metrics=False
 ----
+
+[[openshift-prometheus]]
+== Prometheus on OpenShift
+
+Prometheus is a stand-alone, open source systems monitoring and alerting
+toolkit. You can use Prometheus to visualize metrics and alerts for OpenShift
+system resources.
+
+[IMPORTANT]
+====
+The Prometheus is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+[openshift-prometheus-roles]
+=== Setting Prometheus Role Variables
+
+The Prometheus role creates:
+
+* OpenShift-Prometheus namespace.
+* Prometheus clusterrolebinding and service account.
+* PVs for Prometheus, alertmanager and alertbuffer for internal NFS.
```

Comment:
Do we need to say NFS here? I could see how a customer might interpret this as needing a RWX volume, and ask if EFS or some other volume like gluster (CNS/CRS) might work. 

Is there a better way to phrase this, or expand on what we mean by this?

Reply:
There is no need to mention it at all IMO.  `PVs for Prometheus, alertmanager and alertbuffer`  ... end of bullet